test/controllers/tradingName/WhatToDoControllerSpec.scala

```diff
@@ -146,8 +146,8 @@ class WhatToDoControllerSpec extends ControllerBaseSpec {
           status(result) shouldBe Status.SEE_OTHER
         }
 
```

Comment:
As above in this spec, there are a couple of places session values are added but we don't have tests for them. Could you add them in here as well please